blueprints/cloud-operations/network-dashboard/variables.tf

```diff
@@ -81,3 +81,13 @@ variable "schedule_cron" {
   description = "Cron format schedule to run the Cloud Function. Default is every 10 minutes."
   default     = "*/10 * * * *"
 }
+
+variable "custom_cloud_function_sa" {
+  description = "Preprovisioned custom SA. If one is provided, no new SA will be created."
+  default = ""
+}
+
+variable "custom_scheduler_sa" {
+  description = "Preprovisioned custom SA. If one is provided, no new SA will be created."
+  default = ""
+}
```

Comment:
Can you make this a single variable with `default=null`? If not provided, you create the SA, otherwise you use the one provided.

Reply:
yes sure, however, there was a recent change to the main.tf to add the creation of a second SA for the scheduler, before, both the Cloud Function and the Scheduler for the Network Dashboard used the same SA.

Reply:
I think that change is unrelated. The ask here is two consolidate the two variables into a single one. As far as I can tell you don't need two variables to trigger the creation of the SA

Reply:
Ignore me, apparently I can't read